src/main/java/codeu/model/data/GroupChat.java

```diff
@@ -0,0 +1,51 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.List;
+import java.util.UUID;
+
+public class GroupChat extends Conversation{
+    public HashSet<User> members;
+    public User leader;
+
+    public GroupChat(UUID id, UUID ownerID, User owner, String title, Instant creation) {
```

Comment:
Why are we using both ownerID and owner? Conversation.java uses ownerID but why owner

Reply:
I just did this to streamline the process so we dont have to search for the user by UUID each time and instead we can add the groupchat to the owner obj directly bc I assumed when we call this create function that we would have the user obj already. I also did that bc I made the groupchat field have a owner/leader field ... and I thought having access to the user directly would help. If we do not need those fields as the User obj directly I can change it.

Reply:
if we do not need both that ofc we can remove one of them! but I do feel that having the User obj directly reduces the amount of times we need to write the findUser by UUID method within each class.